.clj-kondo/src/hooks/common.clj

```diff
@@ -407,3 +407,17 @@
 
   ;; should be 1
   (format-string-specifier-count "%-02d"))
+
+(defn add-lsp-ignore-unused-public-var-metadata
```

Comment:
We were doing this in a few places and it was wrong (Kondo `--debug` was super helpful here and helped me figure this out), so I made a helper function so I didn't have to fix like 5 hooks.